configure.ac

```diff
@@ -2388,7 +2397,7 @@ fi
 
 if test "x$with_libdpdk" = "xyes"; then
   SAVE_LDFLAGS="$LDFLAGS"
-  LDFLAGS="$LIBDPDK_LDFLAGS $LDFLAGS"
+  LDFLAGS="$LIBDPDK_LDFLAGS $LIBDPDK_LIBS $LDFLAGS"
```

Comment:
LIBS don't belong in LDFLAGS, but unfortunately that's what the pkgconfig macro returns. I don't think setting them like this has the desired effect, AC_CHECK_LIB still checks just libdpdk for rte_eal_init.